baseplate/observers/tracing.py

```diff
@@ -539,18 +541,26 @@ class TraceQueueFullError(Exception):
 
 
 class SidecarRecorder(Recorder):
-    """Interface for recording spans to a POSIX message queue.
+    """Interface for recording spans to a message queue.
 
     The SidecarRecorder serializes spans to a string representation before
     adding them to the queue.
     """
 
-    def __init__(self, queue_name: str):
-        self.queue = MessageQueue(
-            "/traces-" + queue_name,
-            max_messages=MAX_QUEUE_SIZE,
-            max_message_size=MAX_SPAN_SIZE,
-        )
+    def __init__(self, queue_name: str, queue_type: QueueType = QueueType.POSIX):
```

Comment:
You should either pass the queue object in here directly or optionally accept the host and port for the remote queue

Reply:
accepting the host & port makes sense. Similar to my comment for events above, I dont think we want to make users create the queue themselves